community-operators/spinnaker-operator/spinnaker-operator.v1.13.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,150 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Basic Install
+    categories: "Integration & Delivery"
+    certified: "false"
+    containerImage: docker.io/opsmx11/spinnaker-operator:v0.1
```

Comment:
This is a different image, but the same organization namespace as another "Spinnaker" operator which we accepted already: https://github.com/operator-framework/community-operators/blob/master/community-operators/opsmx-spinnaker-operator/opsmx-spinnaker-operator.v1.13.1.clusterserviceversion.yaml#L8
Are these two operators distinct?

Reply:
These two operators are distinct, as the current one includes only the OpenSource Spinnaker components where as the previous one includes OpsMx components as value added extensions.